TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -191,6 +255,29 @@ namespace Plugin {
     uint32_t TextToSpeechImplementation::SetConfiguration(const Exchange::ITextToSpeech::Configuration &object,Exchange::ITextToSpeech::TTSErrorDetail &ttsStatus)
     {
         TTS::Configuration config;
+        ttsStatus = (Exchange::ITextToSpeech::TTSErrorDetail) Exchange::ITextToSpeech::TTS_INVALID_CONFIGURATION;
+
+        if (!object.ttsEndPoint.empty() && !InputValidation::Instance().validate("ttsendpoint", object.ttsEndPoint))
```

Comment:
These empty value check can be included in the regexp check / validator itself and this part can be simpler

Reply:
Since, writing regrex for empty string, would not be used for all for example: speechrate. Still, we need to check for the other cases.